### DISCUSSION TITLE
logout in dialog and menu of lxqt-panel  on labwc(not lxqt with labwc) didn't work

### DISCUSSION THREAD

**Discussion by ffs-akl** [lxqt/lxqt#2618] [Q&A] [answered]
I used the lxqt-panel in labwc(not lxqt with labwc),  when I wanted to logout, I found that the lxqt-leave worked fine except for "lxqt-leave  --logout"   It looked like the icon of logout is grayed out and not clickable.
 I'm looking forward to your help to get the logout work.

**Comment by stefonarch** [comment, marked as answer]
>I used the lxqt-panel in labwc(not lxqt with labwc)

You cannot logout the lxqt-session if you didn't start it.

**Reply by ffs-akl** [reply]
I see, the logout command of labwc is` labwc -exit`,  so logout on dialog is no way to work.  
thank you very much!
